app/models/user.rb

```diff
@@ -159,11 +160,15 @@ def self.upload_user_list(user_class, user_list, encoding)
           # don't know how to fetch line so we concat given array
           next if CSV.generate_line(row).strip.empty?
           if processed_users.include?(row[0])
-            result[:invalid_lines] = I18n.t('csv_upload_user_duplicate',
-                                            {user_name: row[0]})
+            if result[:invalid_lines].count < max_invalid_lines
+              result[:invalid_lines] << I18n.t('csv_upload_user_duplicate',
+                                              {user_name: row[0]})
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.